backupwordpress.php

```diff
@@ -6,7 +6,7 @@
 Version: 3.3 Alpha
 Author: Human Made Limited
 Author URI: http://hmn.md/
-License: GPL-2.0+
+License: GPL-3.0
```

Comment:
JetPack simply goes with `GPL2+`: https://github.com/Automattic/jetpack/blob/master/jetpack.php#L10, I vote we switch to that.